content/en/_includes/page-drifted-msg.md

```diff
@@ -17,7 +19,11 @@ The content of this page may be <b>outdated</b> and some links may be invalid.
     and search for <code>{{$def_lang_path}}</code>.
   </p>
 </details>
+
```

Comment:
We don't need an extra blank line, because the `{{ end }}` directive is already on it's own line.

```suggestion
```